### PR TITLE
Ports AI Portrait Picker + Curator Portrait Picker [PORT] [BOUNTY]

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation
Ports https://github.com/tgstation/tgstation/pull/59146
Ports https://github.com/tgstation/tgstation/pull/53994

### Intent of your Pull Request

Pretty, see original PRs for additional info

image
image

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
See UI pictures
Access AI core thing by picking the Question mark Core Icon in the selection.

### What should players be aware of when it comes to the changes your PR is implementing?

# Changelog

:cl:  tralezab
rscadd: Curators can print paintings  
rscadd: AIs can pick paintings as their icon
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Theres conflict with donator privileges here where custom ai portraits ar largely reserved to donors

**Comment by Bibby0110** [comment]
> Theres conflict with donator privileges here where custom ai portraits ar largely reserved to donors

what are you on about
the only thing being donor gives you is the donor + personal skins.

anybody can select a different AI core image, donators get the donator ones in addition.

**Comment by alexkar598** [comment]
There still exists overlap, ie skins that would be normally unobtainable become now obtainable by making them paintings + anyone can create a custom skin and use it

**Comment by Bibby0110** [comment]
> There still exists overlap, ie skins that would be normally unobtainable become now obtainable by making them paintings + anyone can create a custom skin and use it

Then moderate the content? lol this is perhaps the weakest excuse for shitting on this PR.

You can't recreate the animated skins either.

**Comment by ToasterBiome** [comment]
How about making the custom ai portraits a donator only feature

**Comment by Bibby0110** [comment]
> How about making the custom ai portraits a donator only feature

1. Not going to monetize someone elses work without their permission
2. Don't like donator only features

**Comment by ToasterBiome** [comment]
> > How about making the custom ai portraits a donator only feature
> 
> 1. Not going to monetize someone elses work without their permission
> 2. Don't like donator only features

Fair enough

**Comment by patpol4** [comment]
What's up with the map changes

**Comment by Bibby0110** [comment]
> What's up with the map changes

Adds the console for the curator and map merged the maps

**Comment by Bibby0110** [comment]
> What's up with the map changes

nevermind managed to fuck that up, should be good now

**Review by nesisknesis — APPROVED** [review]
this can range from quite good to absolutely abysmal

**Comment by Bibby0110** [comment]
> this can range from quite good to absolutely abysmal

the only way this goes wrong is if the admins don't remove rule breaking stuff

**Comment by maxion12345** [comment]
either 
A) doesn't get merged because muh donator
B) doesn't get merged for fear of CockBot V2.5 with a penis portrait
C) gets merged and we enjoy it

**Comment by Bibby0110** [comment]
> either
> A) doesn't get merged because muh donator
> B) doesn't get merged for fear of CockBot V2.5 with a penis portrait
> C) gets merged and we enjoy it

yep

**Review by patpol4 — APPROVED** [review]
Map appears to be fine. sad to see a plant go but its for a good cause.

**Comment by Bibby0110** [comment]
> Map appears to be fine. sad to see a plant go but its for a good cause.

plant o7
